src/seedu/addressbook/data/person/UniquePersonList.java

```diff
@@ -81,6 +82,23 @@ public UniquePersonList(UniquePersonList source) {
         return Collections.unmodifiableList(internalList);
     }
 
+    /**
+     * Returns an unmodifiable java List view sorted in alphabetical order
+     * with elements cast as immutable {@link ReadOnlyPerson}
+     */
+    public List<ReadOnlyPerson> sortImmutableListView() {
+        final List<ReadOnlyPerson> sortedList = new ArrayList<>();
+        sortedList.addAll(internalList);
+        sortedList.sort(new Comparator<ReadOnlyPerson>() {
+            @Override
```

Comment:
Perhaps can abstract the compare so that comparison functions can be passed to sort to reuse the function to sort the list in multiple ways